src/index.ts

```diff
@@ -87,6 +87,28 @@ const createWindow = (): void => {
 
   mainWindow.webContents.once('did-attach-webview', (event, webContents) => {
     webview = webContents;
+
+    const isInAppUrl = (url: string) =>
+      url.startsWith(`npld-viewer://${process.env.NPLD_PLAYER_PREFIX}`);
```

Comment:
I think we want it to just be this:

```suggestion
      url.startsWith(process.env.NPLD_PLAYER_PREFIX);
```

my understanding is that the `npld-viewer://` links would only appear when not already browsing in the viewer (eg. when the auth header is missing)